media-libs/amdgpu-pro-vulkan/amdgpu-pro-vulkan-21.50.1.1376756.ebuild

```diff
@@ -30,7 +30,9 @@ RESTRICT="bindist mirror"
 LICENSE="AMD-GPU-PRO-EULA"
 SLOT="0"
 KEYWORDS="-* ~amd64 ~x86"
-IUSE="abi_x86_32 abi_x86_64"
+IUSE="abi_x86_32 abi_x86_64 video_cards_amdgpu"
+
+REQUIRED_USE="video_cards_amdgpu"
```

Comment:
While this isn't wrong, I wonder if this is right either. Maybe you just want to depend on x11-drivers/xf86-video-amdgpu?

Reply:
I want to say that this package only works on systems running on an AMD GPU whose driver is `amdgpu`. I asked in the dev IRC channel and I got suggested this solution and I liked it.

> Maybe you just want to depend on x11-drivers/xf86-video-amdgpu?

I don't want to pull it in particular, although maybe it's needed ?